src/main/proto/com/google/api/codegen/config.proto

```diff
@@ -314,3 +318,36 @@ message BundlingDescriptorProto {
   // together.
   string subresponse_field = 3;
 }
+
+// ExperimentalFeatures contain special-case features that,
+// for lack of similar problems, we don't quite know how to properly
+// expose as a config.
+// Configs defined here should eventually be replaced with more general ones.
+message ExperimentalFeatures {
+  repeated IamResourceProto iam_resources = 1;
+}
+
+message IamResourceProto {
```

Comment:
add a comment saying that this is experimental - even though it is already in ExperimentalFeatures, it might get moved around / separated later.